Simplenote/src/main/java/com/automattic/simplenote/InfoBottomSheetDialog.java

```diff
@@ -136,6 +137,11 @@ private String getWordCount(String content) {
         return String.format("%s %s", formattedWordCount, wordCountString);
     }
 
+    private String getCharactersCount(String content) {
+        int numChars = content.length();
+        String charCountString = mFragment.getResources().getQuantityString(R.plurals.char_count,numChars);
+        return String.format("%s %s", numChars , charCountString);
```

Comment:
Formatting: `numChars,` ...

Reply:
Does this count spaces? Should it count spaces? :)

Reply:
<img width="1161" alt="screen shot 2018-07-24 at 10 27 01" src="https://user-images.githubusercontent.com/3122461/43113268-4a787748-8f2c-11e8-9135-98e21aab7ed4.png">
<img width="624" alt="screen shot 2018-07-24 at 10 28 58" src="https://user-images.githubusercontent.com/3122461/43113346-9a50b1b8-8f2c-11e8-97ae-6bf4064f1eb6.png">

it counts spaces on other clients, so I think it may work like other clients 👍

Reply:
ok thanks for checking on that.